client/src/pages/Home/Forms/LaptopRequest/LaptopForm.js

```diff
@@ -51,15 +51,32 @@ function LaptopForm() {
 
 		return true;
 	}
+	const closeModale = document.querySelector(".close_modale");
+	const modale = document.querySelector(".modale-content");
+	const form = document.querySelector(".form");
+	//function display greeting after submit the from
+
 	function messageAlert() {
-		if (validateForm(document.laptopRequestForm)) {
-			alert(
-				"thank you for your completing the request form. your request have been recived "
-			);
-		}
+		console.log("heloo");
+		document.body.style.backgroundColor = "black";
+		modale.style.display = "block";
```

Comment:
This is not OK: you may not modify the DOM that is rendered by the compo directly in React.

Reply:
i got you i will take it out,